src/components/table.ts

```diff
@@ -81,6 +81,20 @@ export module Components {
     }
 
     /**
+     * Returns the Component at the specified row and column index.
+     *
+     * @param {number} rowIndex
+     * @param {number} columnIndex
+     * @returns {Component} The Component at the specified position, or null if no Component is there.
+     */
+    public componentAt(rowIndex: number, columnIndex: number) {
+      if (rowIndex < 0 || rowIndex >= this._nRows || columnIndex < 0 || columnIndex >= this._nCols) {
+        return null;
+      }
+      return this._rows[rowIndex][columnIndex];
+    };
```

Comment:
You mean `add()`? Or something like that? There might not be a way to name it that makes it clear this is a getter under our chosen conventions.